akka-stream/src/main/scala/akka/stream/ActorFlowMaterializer.scala

```diff
@@ -152,13 +151,20 @@ abstract class FlowMaterializer {
    */
   def withNamePrefix(name: String): FlowMaterializer
 
-  // FIXME this is scaladsl specific
   /**
    * This method interprets the given Flow description and creates the running
    * stream. The result can be highly implementation specific, ranging from
    * local actor chains to remote-deployed processing networks.
    */
-  def materialize[Mat](runnable: RunnableFlow[Mat]): Mat
+  def materialize[Mat](runnable: Graph[ClosedShape, Mat]): Mat
```

Comment:
Users will use the DSL to materialize things, not this bare-bones facility. And this is the only sensible way to express a generic “closed graph” concept without huge API baggage—the problem with RunnableFlow is that it has a language-specific `mapMaterialized` method on it.